sympy/solvers/solveset.py

```diff
@@ -1136,24 +1136,24 @@ def _transolve(f, symbol, domain):
 
     Solving helpers: These are the helpers that solve the equation.
```

Comment:
-> Solving helpers: once an equation is identified, a corresponding helper either solves the equation or returns a form of the equation that `sovleset` might better be able to handle.